store/keys.go

```diff
@@ -8,7 +8,10 @@ import (
 	"github.com/celestiaorg/go-header"
 )
 
-var headKey = datastore.NewKey("head")
+var (
+	headKey = datastore.NewKey("head")
+	tailKey = datastore.NewKey("tail")
```

Comment:
discussion:
technically they doesn have to be separate value and could be unified, but changing that is breaking

Reply:
1) Do you mean to store 2 values under 1 key?
2) What is the point to unify under a 1 key?

Reply:
Yes
Actually not much